app/models/ping_thing.rb

```diff
@@ -39,7 +39,11 @@ class PingThing < ApplicationRecord
   validates_length_of :application, maximum: 80, allow_blank: true
   validates :network, inclusion: { in: %w(mainnet testnet) }
   validates :signature, length: { in: 64..128 }
- 
+
+  scope :for_date_range_and_network, -> (network, from, to) {
```

Comment:
Maybe it'd be better to use reported_at in the scope name so it's more descriptive like `for_reported_at_range_and_network`